std/assembly/map.ts

```diff
@@ -73,14 +73,12 @@ export class Map<K,V> {
   private entriesOffset: i32 = 0;
   private entriesCount: i32 = 0;
 
-  constructor(initialEntries: MapInitialEntries<K,V> = []) {
-    let entriesLength = initialEntries.length;
```

Comment:
plz revert this cached entriesLength var and use instead `initialEntries.length`